include/gz/sensors/config.hh.in

```diff
@@ -1,20 +1,42 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 /* Config.hh. Generated by CMake for @PROJECT_NAME_NO_VERSION@. */
 
+#ifndef GZ_SENSORS_CONFIG_HH_
+#define GZ_SENSORS_CONFIG_HH_
+
 /* Version number */
-#define IGNITION_SENSORS_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
-#define IGNITION_SENSORS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
-#define IGNITION_SENSORS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+#define GZ_SENSORS_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
+#define GZ_SENSORS_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_SENSORS_PATCH_VERSION ${PROJECT_VERSION_PATCH}
 
-#define IGNITION_SENSORS_VERSION "${PROJECT_VERSION}"
-#define IGNITION_SENSORS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_SENSORS_VERSION "${PROJECT_VERSION}"
+#define GZ_SENSORS_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_SENSORS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
+#define GZ_SENSORS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
 
-#define IGNITION_SENSORS_VERSION_HEADER "Ignition ${IGN_DESIGNATION_CAP}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2014 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define GZ_SENSORS_VERSION_HEADER "Gz Sensors, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2014 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
 
 #cmakedefine BUILD_TYPE_PROFILE 1
 #cmakedefine BUILD_TYPE_DEBUG 1
 #cmakedefine BUILD_TYPE_RELEASE 1
 
-#define IGN_SENSORS_PLUGIN_PATH "@IGN_SENSORS_PLUGIN_PATH@"
-#define IGN_SENSORS_PLUGIN_NAME(name) std::string("@PROJECT_LIBRARY_TARGET_NAME@-")+name
+#define GZ_SENSORS_PLUGIN_PATH "@IGN_SENSORS_PLUGIN_PATH@"
```

Comment:
This would need to be set here:

https://github.com/gazebosim/gz-sensors/blob/75027c2d6d2b5c8b28b8d2228f140737a4c3c96e/CMakeLists.txt#L94

But I actually think this can be removed on Garden because sensors aren't plugins anymore since 

* #90